### PR TITLE
{bio}[foss/2022b] Infernal v1.1.4

### PR DESCRIPTION
**Pull request by Ghepardo**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/i/Infernal/Infernal-1.1.4-foss-2022b.eb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+easyblock = 'ConfigureMake'
```
Comment (verdurin) [inline easybuild/easyconfigs/i/Infernal/Infernal-1.1.4-foss-2022b.eb:1]: @Ghepardo could you restore the attribution and license information from previous versions please?

**Review by verdurin — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/i/Infernal/Infernal-1.1.4-foss-2022b.eb (hunk 1 above)
Comment (Ghepardo) [reply easybuild/easyconfigs/i/Infernal/Infernal-1.1.4-foss-2022b.eb:14]: Of course - I've submitted an update to this PR.  (I must admit I didn't have a lot of faith in those initial comment blocks generally - other contributors seem to have copied them blindly for certain new easyconfigs.)

Comment (verdurin) [reply easybuild/easyconfigs/i/Infernal/Infernal-1.1.4-foss-2022b.eb:14]: Thanks.

**Comment by verdurin** [comment]
@boegelbot please test @ jsc-zen2

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/e3211b21291894fd42be4c29bd27281b for a full test report.

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
Going in, thanks @Ghepardo!

**Pull request merged**